### PR TITLE
#2543: Org feature org pages review - [BOB] - MIGRATIONS

### LINKED ISSUE #2543: Design Review: View and manage organization Details
### Issue description

In the org model, all org users should be able see information about their organization. Admins can change their organization, but no one else can. No org user can change the Senior official in these pages. 

Design should review and compare the output for the organization details and compare against the most recent figma and requirements. 

### Acceptance criteria

- [ ] Design reviews the organization details (mailing address and senior official)
- [ ] Document any changes needed or new additions in a document, including with screenshots so it is easy to follow
- [ ] Discuss changes with dev and determine which are small enough to be done as part of this ticket
- [ ] Work with dev to create new tickets for larger changes

### Additional context

**Working with dev:** Dev effort on this shouldn't be more than a "2" (2 full work days).

- Dev is responsible for answering questions from design, making minor changes (such as those that we would normally make on a PR).
- Anything that would take more than 4 hours to complete alone should be it's own ticket.
- If the amount of changes desired would result in the dev spending more than a full day on it, make a dev ticket for items where reasonable (communicate with Alysia).

NOTE: This design review does not include testing permissions and/or sending emails. That will be tested separately once that work is implemented.

### Temporary Slack channel
Discussions for this design review should take place in this Slack channel: [#2543-design-review](https://dhscisa.enterprise.slack.com/archives/C07KZQZBTTJ)

### Links to other issues

- blocked by #2378

### PR DESCRIPTION
**Pull request by rachidatecs**
## Ticket

Resolves #2543

## Changes

- Fix main nav nav active state when on senior official page
- Margin tweak between required msg and form
- Refactor placement of success and error messages on portfolio pages (msg does not extend over sidenav)
- All org name compoenents title change (remove 'and mailing address')
- State / territory labels change state, territory, or military post
- Some clean up of button scss

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the associated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [ ] Checked that the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

File: src/registrar/templates/portfolio_domains.html [hunk 1]
```diff
@@ -9,6 +9,10 @@
 {% endblock %}
 
 {% block portfolio_content %}
+{% block messages %}
+  {% include "includes/form_messages.html" %}
+{% endblock %}
+
 <div id="main-content">
```
[the remark this answers is not in the document]
Comment (rachidatecs) [reply src/registrar/templates/portfolio_domains.html:16]: Incorrect. Most of these are outside the main content, the org one is inside the main content to position it beside the sidenav (not on top)

File: src/registrar/templates/includes/senior_official.html [hunk 2]
```diff
@@ -35,15 +35,17 @@ <h1>Senior official</h1>
 {% elif not form.full_name.value and not form.title.value and not form.email.value %}
     <h4>No senior official was found.</h4>
```
[the remark this answers is not in the document]
Comment (rachidatecs) [reply src/registrar/templates/includes/senior_official.html:36]: I don't think so. This is just copy, the other 2 instances are blocks of content.

**Review by gabydisarli — APPROVED** [review]
Design is approved ✅

File: src/registrar/assets/sass/_theme/_buttons.scss [hunk 3]
```diff
@@ -1,18 +1,23 @@
...
+// Used on: NONE
 a[href$="todo"]::after {
   background-color: yellow;
   color: color(blue-80v);
   content: " [link TBD]";
   font-style: italic;
 }
 
+// Used on: profile
+// Note: Is this needed?
```
Comment (abroddrick) [inline src/registrar/assets/sass/_theme/_buttons.scss:14]: what happens if you remove it?

File: src/registrar/assets/sass/_theme/_buttons.scss [hunk 4]
```diff
@@ -94,10 +120,22 @@ a.usa-button--outline:active {
...
+a.withdraw:focus {
+  background-color: color('error-dark');
+}
   
+a.withdraw:active {
+  background-color: color('error-darker');
+}
+
+// Used on: Domain request status
+//NOTE: Revise to BEM convention usa-button--outline-secondary
```
Comment (abroddrick) [inline src/registrar/assets/sass/_theme/_buttons.scss:138]: Save me referencing BEM, if this isn't up to the convention, note it in a refactor ticket?

**Review by abroddrick — APPROVED** [review]
I see a lot of code on how this could be made better or may include scss that isn't used. Can you make a ticket for refactoring these files (you can just call out the file names or  link the PR). It seems worth while for us to just come back and clean this up. Love the comments you added though! Ignore my other comment about making a ticket for the one line I commented on. One ticket for refactoring these all and bringing up to standards would be ideal

**Comment by rachidatecs** [comment]
@abroddrick it's a matter of time before I'm working on some buttons again (or helping/reviewing work on buttons css). I'd do the cleanup then. I don't think we need a ticket.

The commenting/research was done to answer Gaby's questions as she works on updating her Figma design system.

**Pull request merged**